### PR TITLE
Adding Swarnima Gautam as contestant

### PR DESCRIPTION
**Pull request by Swarnima-27**

### CODE REVIEW

File: contestants.json [hunk 1]
```diff
@@ -343,6 +343,14 @@
       "facebook" : "https://www.facebook.com/anshumanv12",
       "github" : "https://github.com/anshumanv",
       "twitter" : "https://twitter.com/Anshumaniac12"
+    },
+    {
+      "name" : "Swarnima Gautam",
+      "imageur1" : "/contestant/pic.jpg",
+      "about" : "learning opencode",
+      "facebook" : "https://www.facebook.com/swarnima.gautam.14"
```
Comment (pradeepgangwar) [inline contestants.json]: @Swarnima-27  please add `,` at the end of this line and rename your image.

File: contestants.json [hunk 2]
```diff
@@ -343,6 +343,14 @@
       "facebook" : "https://www.facebook.com/anshumanv12",
       "github" : "https://github.com/anshumanv",
       "twitter" : "https://twitter.com/Anshumaniac12"
-    }
+    },
+       {
+          "name" : "Swarnima Gautam",
+          "imageur1" : "/contestant/swarnima.jpg",
+          "about" : "learning opencode",
+          "facebook" : "https://www.facebook.com/swarnima.gautam.14"
```
Comment (pradeepgangwar) [inline contestants.json]: Add `,` at the end of this line and delete image with `pic.jpg` name.

**Comment by pradeepgangwar** [comment]
I fixed many of your changes this time. Next time be cautious.

**Comment by Swarnima-27** [comment]
okk sir

**Pull request merged**